apps/aehttpserver/lib/aehttpserver/web/controllers/oracle_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Aehttpserver.Web.OracleController do
+  use Aehttpserver.Web, :controller
+
+  alias Aecore.OraclePrototype.Oracle
+  alias Aeutil.Bits
+
+  require Logger
+
+  def oracle_response(conn, _params) do
+    body = conn.body_params
```

Comment:
`/oracle_response` should have some authorization as it's visible to the world. (You can for example check if the response come from localhost, that should be good enough for now)

Reply:
as far as I can see, this is addressed already